alpine/Dockerfile

```diff
@@ -0,0 +1,27 @@
+ARG ARCH=""
+FROM        ${ARCH}alpine:latest as alpine
+
+RUN apk upgrade -U && \
+    apk add tzdata ca-certificates
+
+# Use the busybox.static to avoid dynamic library dependencies.
+RUN apk add busybox-static && \
+    mv /bin/busybox.static /bin/busybox && \
+    /bin/busybox --install -s /bin
+
```

Comment:
Previous iterations were trying to not include musl-libc .  In this iteration, I haven't remove it.  If that is okay, then we can leave the default dynamic linked `busybox`.  

```suggestion
```